life/modernized/27-chapter-27.adoc

```diff
@@ -5971,9 +6218,14 @@ contributed in no inconsiderable degree, to that salutary change.
 
 The following epistle, written about this time, is worthy of a place in these memoirs;
 
+[.embedded-content-document.epistle]
+--
+
+[.letter-heading]
 For the Friends, brethren and sisters of the Yearly Meeting at Bristol,
 for this present year.
 
+[.signed-section-context-open]
 Worminghurst, 4th of Third month, 1699.
 
 My dearly beloved friends and brethren in the Lord.--Dear and precious
```

Comment:
Lets take this first sentence (My dearly beloved friends and brethren in the Lord,) and make it a [.salutation]. 👍🏻